### PR TITLE
Call method without arguments or empty object

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
* Allow method calling without required argument for non-streaming and server-streaming methods

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (alexander-fenster) [inline .idea/.gitignore]: We normally don't checkin editor-specific files. You can add the whole `.idea` folder to `.gitignore`.

Comment (alexander-fenster) [reply .idea/.gitignore]: The `.DS_Store` files should not be checked in either.

**Comment by alexander-fenster** [comment]
`.DS_Store` files are still there, can you please delete them as well? Thank you!

File: .idea/.gitignore (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply .idea/.gitignore]: Sure, done.

**Pull request merged**